src/components/pages/coach/CoachDashboard/index.jsx

```diff
@@ -24,11 +38,508 @@ import { ReactComponent as MorningImg } from  '../../../../assets/icons/morning.
+                        );
+                        console.log(
+                            user.address,
+                            customer.default_address.address1,
+                            user.address === customer.default_address.address1
+                        );
+                        console.log(
+                            user.firstName,
+                            customer.first_name,
+                            user.firstName === customer.first_name
+                        );
+                        console.log(
+                            user.lastName,
+                            customer.last_name,
+                            user.lastName === customer.last_name
+                        );
+                        console.log(
+                            user.city,
+                            customer.default_address.city,
+                            user.city === customer.default_address.city
+                        );
+                        console.log(
+                            user.postalCode,
+                            customer.default_address.zip,
+                            user.postalCode === customer.default_address.zip
+                        );
+                        console.log(
+                            user.country,
+                            customer.default_address.country_code,
+                            user.country === customer.default_address.country_code
+                        );
+                        console.log(
+                            user.telephone,
+                            customer.phone,
+                            user.telephone === customer.phone
+                        );
+                        console.log(
+                            user.newsletter,
+                            customer.accepts_marketing,
+                            user.newsletter === customer.accepts_marketing
+                        );*/
+                        
+                        this._modifyUser(user.id, customer);
+                    }
+                }
+            })
+        });
+        console.log("Total synced users: "+t);
+
+        this.setState({
+            syncing: undefined
+        });
+    }
+
+    _modifyUser = (id, customer) => {
+        // Get all values and prepare them for API handling
+        /*this.props.alter({
+            variables: {
+                id,
+                address: customer.default_address.address1,
+                city: customer.default_address.city,
+                email: customer.email,
+                firstname: customer.first_name,
+                lastname: customer.last_name,
+                phone: customer.phone,
+                zip: customer.default_address.country_code,
+                country: customer.default_address.country_code
+            }
+        }).then(({data}) => {
+            console.log("Success",data);
+        })
+        .catch((error) => {
+            console.log("Error",error);
```

Comment:
# please remove `console` (no-console)

In JavaScript that is designed to be executed in the browser, it's considered a best practice to avoid using methods on `console`. Such messages are considered to be for debugging purposes and therefore not suitable to ship to the client. In general, calls using `console` should be stripped before being pushed to production.

```js
console.log("Made it here.");
console.error("That shouldn't have happened.");
```

Examples of **incorrect** code for this rule:

```js
/* eslint no-console: "error" */

console.log("Log a debug level message.");
console.warn("Log a warn level message.");
console.error("Log an error level message.");
```

Reply:
Noted.
However, the console functions do not exist without reason. It is a really great, fully integrated development tool.